.yarnrc.yml

```diff
@@ -24,18 +24,35 @@ packageExtensions:
   "@material-ui/lab@^4.12.3":
     dependencies:
       "@material-ui/types": ^5.1.0
-  "@types/express-serve-static-core@*":
-    peerDependencies:
+  "@types/http-proxy@*":
```

Comment:
These changes are to silence warnings on startup from Yarn PNP. :sob:

Reply:
Holy fuck you did it